include/vcpkg/commands.generate-message-map.h

```diff
@@ -1,10 +1,20 @@
 #include <vcpkg/base/files.h>
+#include <vcpkg/base/messages.h>
 
 #include <vcpkg/commands.interface.h>
 #include <vcpkg/vcpkgcmdarguments.h>
 
 namespace vcpkg::Commands
 {
+    struct FormatArgMismatches
+    {
+        std::vector<StringView> arguments_without_comment;
+        std::vector<StringView> comments_without_argument;
+    };
+
+    std::vector<StringView> get_all_format_args(StringView fstring, msg::LocalizedString& error);
```

Comment:
Note: this doesn't use the Parser stuff, since it's all-internal and doesn't need to have good errors.